### PR TITLE
fix checkstyle core-util issue

### PR DESCRIPTION
**Pull request by seagle-yuan**
fix checkstyle issue

com.baidu.hugegraph.traversal.algorithm.HugeTraverser.java
com.baidu.hugegraph.traversal.algorithm.JaccardSimilarTraverser.java
com.baidu.hugegraph.traversal.algorithm.OltpTraverser.java
com.baidu.hugegraph.traversal.algorithm.ShortestPathTraverser.java
com.baidu.hugegraph.traversal.algorithm.records.ShortestPathRecords.java
com.baidu.hugegraph.traversal.optimize.HugeScriptTraversal.java
com.baidu.hugegraph.traversal.optimize.TraversalUtil.java
com.baidu.hugegraph.type.HugeType.java
com.baidu.hugegraph.type.define.SerialEnum.java
com.baidu.hugegraph.util.ConfigUtil.java
com.baidu.hugegraph.util.Consumers.java
com.baidu.hugegraph.util.GZipUtil.java
com.baidu.hugegraph.util.KryoUtil.java
com.baidu.hugegraph.util.Reflection.java
com.baidu.hugegraph.util.collection.IntMap.java
com.baidu.hugegraph.util.collection.IntSet.java

com.baidu.hugegraph.unit.core.SerialEnumTest

### CODE REVIEW

File: hugegraph-core/src/main/java/com/baidu/hugegraph/util/Reflection.java [hunk 1]
```diff
@@ -55,39 +55,39 @@ public class Reflection {
...
+                        REFLECTION_CLAZZ.getMethod("registerFieldsToFilter",
                                           Class.class, String[].class);
             } catch (Throwable e) {
                 LOG.error("Can't find registerFieldsToFilter method", e);
             }
 
             try {
                 registerMethodsToFilterMethodTemp =
-                reflectionClazz.getMethod("registerMethodsToFilter",
+                        REFLECTION_CLAZZ.getMethod("registerMethodsToFilter",
```
Comment (javeme) [inline hugegraph-core/src/main/java/com/baidu/hugegraph/util/Reflection.java:71]: also update alignment

File: hugegraph-core/src/main/java/com/baidu/hugegraph/util/Reflection.java [hunk 2]
```diff
@@ -55,39 +55,39 @@ public class Reflection {
...
+        if (REGISTER_FILEDS_TO_FILTER_METHOD == null) {
             throw new NotSupportException(
                       "Reflection.registerFieldsToFilter()");
         }
 
         try {
-            registerFieldsToFilterMethod.setAccessible(true);
-            registerFieldsToFilterMethod.invoke(reflectionClazz,
+            REGISTER_FILEDS_TO_FILTER_METHOD.setAccessible(true);
+            REGISTER_FILEDS_TO_FILTER_METHOD.invoke(REFLECTION_CLAZZ,
```
Comment (javeme) [inline hugegraph-core/src/main/java/com/baidu/hugegraph/util/Reflection.java:90]: also update alignment

File: hugegraph-core/src/main/java/com/baidu/hugegraph/util/Reflection.java [hunk 3]
```diff
@@ -98,14 +98,14 @@ public static void registerFieldsToFilter(Class<?> containingClass,
...
+        if (REGISTER_METHODS_TO_FILTER_MOTHOD == null) {
             throw new NotSupportException(
                       "Reflection.registerMethodsToFilterMethod()");
         }
 
         try {
-            registerMethodsToFilterMethod.setAccessible(true);
-            registerMethodsToFilterMethod.invoke(reflectionClazz,
+            REGISTER_METHODS_TO_FILTER_MOTHOD.setAccessible(true);
+            REGISTER_METHODS_TO_FILTER_MOTHOD.invoke(REFLECTION_CLAZZ,
```
Comment (javeme) [inline hugegraph-core/src/main/java/com/baidu/hugegraph/util/Reflection.java:108]: also update alignment

File: hugegraph-core/src/main/java/com/baidu/hugegraph/util/collection/IntMap.java [hunk 4]
```diff
@@ -67,7 +72,7 @@ public static final class IntMapBySegments implements IntMap {
 
         private static final int DEFAULT_SEGMENTS = IntSet.CPUS * 100;
         private static final Function<Integer, IntMap> DEFAULT_CREATOR =
-                             size -> new IntMapByFixedAddr(size);
+            size -> new IntMapByFixedAddr(size);
```
Comment (javeme) [inline hugegraph-core/src/main/java/com/baidu/hugegraph/util/collection/IntMap.java:75]: prefer to keep old style

File: hugegraph-core/src/main/java/com/baidu/hugegraph/util/collection/IntSet.java [hunk 5]
```diff
@@ -58,7 +61,7 @@ public static final class IntSetBySegments implements IntSet {
 
         private static final int DEFAULT_SEGMENTS = IntSet.CPUS * 100;
         private static final Function<Integer, IntSet> DEFAULT_CREATOR =
-                             size -> new IntSetByFixedAddr4Unsigned(size);
+            size -> new IntSetByFixedAddr4Unsigned(size);
```
Comment (javeme) [inline hugegraph-core/src/main/java/com/baidu/hugegraph/util/collection/IntSet.java:64]: ditto

File: hugegraph-core/src/main/java/com/baidu/hugegraph/util/Reflection.java (hunk 1 above)
Comment (javeme) [inline hugegraph-core/src/main/java/com/baidu/hugegraph/util/Reflection.java:71]: also update alignment of line 72

File: hugegraph-core/src/main/java/com/baidu/hugegraph/util/Reflection.java (hunk 2 above)
Comment (javeme) [inline hugegraph-core/src/main/java/com/baidu/hugegraph/util/Reflection.java:90]: also update alignment of line 91

File: hugegraph-core/src/main/java/com/baidu/hugegraph/util/Reflection.java (hunk 3 above)
Comment (javeme) [inline hugegraph-core/src/main/java/com/baidu/hugegraph/util/Reflection.java:108]: also update alignment of line 109

**Comment by javeme** [comment]
@seagle-yuan there are still comments~

File: hugegraph-core/src/main/java/com/baidu/hugegraph/util/Reflection.java [hunk 6]
```diff
@@ -69,7 +69,7 @@ public class Reflection {
             try {
                 registerMethodsToFilterMethodTemp =
                         REFLECTION_CLAZZ.getMethod("registerMethodsToFilter",
-                                          Class.class, String[].class);
+                                                   Class.class, String[].class);
```
Comment (javeme) [inline hugegraph-core/src/main/java/com/baidu/hugegraph/util/Reflection.java:72]: also update alignment of registerFieldsToFilter

**Review by javeme — APPROVED** [review]

**Review by imbajin — APPROVED** [review]

**Pull request merged**